### PR TITLE
AAP-19236 - Add assemblies for Access Management and peer review changes for modules from 1647

### PR DESCRIPTION
**Pull request by dcdacosta**
This PR updates and adds the assembly modules for Access management. It also includes the peer review changes made to PR #1647 .

### CODE REVIEW

File: downstream/assemblies/platform/assembly-controller-teams.adoc [hunk 1]
```diff
@@ -1,24 +1,31 @@
...
+
+= Teams
 
 :context: controller-teams
-= Managing teams
 
-A *Team* is a subdivision of an organization with associated users, projects, credentials, and permissions.
-Teams offer a means to implement role-based access control schemes and delegate responsibilities across organizations.
-For example, you can grant permissions to a whole team rather than to each user on the team.
+A team is a subdivision of an organization with associated users, and resources. Teams provide a means to implement role-based access control schemes and delegate responsibilities across organizations. For instance, you can grant permissions to a Team rather than each user on the team.
```
Comment (hherbly) [inline downstream/assemblies/platform/assembly-controller-teams.adoc:11]: A couple of comments: 

- I would remove the comma after "associated users"--looks like it's left over from the previous version
- Third sentence, looks like there's an errant capitalization -- "you can grant permissions to a Team"

File: downstream/assemblies/platform/assembly-controller-teams.adoc [hunk 2]
```diff
@@ -1,24 +1,31 @@
...
+You can create as many teams as needed for your organization. Teams can only be assigned to one organization while an organization can be made up of multiple teams. Each team can be assigned roles, the same way roles are assigned for users. Teams can also scalably assign ownership for credentials, preventing multiple interface click-throughs to assign the same credentials to the same user.
```
Comment (hherbly) [inline downstream/assemblies/platform/assembly-controller-teams.adoc:12]: Rewrite of the last sentence in this line for clarity:
With teams you can assign credential ownership at scale, preventing the need to click through multiple interfaces to assign the same credentials to each individual user.
(if that's not what you were trying to say please disregard)

File: downstream/assemblies/platform/assembly-gw-resources.adoc [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+:_mod-docs-content-type: ASSEMBLY
+
+[id="assembly-gw-resources_{context}"]
+
+= Resources
+
+:context: access-mgmt-resources
+
+You can manage who has access to {PlatformNameShort} resources and what users can do with those resources. Users are granted access through the roles to which they are assigned or through roles inherited through the role hierarchy, for example, through the roles they inherit through team membership. {PlatformNameShort} resources differ depending on the functionality you are configuring. For example, resources can be job templates and projects for automation execution or decision environments and rulebook activations for automation decisions.
```
Comment (hherbly) [inline downstream/assemblies/platform/assembly-gw-resources.adoc:11]: I'd change "who has access" to "user access"

File: downstream/modules/platform/proc-gw-organizations-exec-env.adoc [hunk 4]
```diff
@@ -2,24 +2,24 @@
...
+When {ControllerName} is enabled on the platform, you can review any {ExecEnvShort}s you have set up and manage their settings within the organization resource.
 
 For more information about execution environments, see link:{BaseURL}/{PlatformVers}/html/using_automation_execution/index#assembly-controller-execution-environments[Execution environments] in _Using automation execution_ guide.
 
 
 .Procedure
 
 . From the navigation panel, select {MenuAMOrganizations}.
-. From the Organizations list view, select the organization to which you want to manage execution environments.
+. From the Organizations list view, select the organization to which you want to manage {ExecEnvShort}s.
```
Comment (hherbly) [inline downstream/modules/platform/proc-gw-organizations-exec-env.adoc:15]: I'd change to "select the organization whose EEs you want to manage."

**Review by hherbly — APPROVED** [review]
Approved, but left a couple of comments. Lmk when you're ready to squash and merge.

File: downstream/assemblies/platform/assembly-controller-teams.adoc (hunk 2 above)
Comment (dcdacosta) [reply downstream/assemblies/platform/assembly-controller-teams.adoc]: Fixed.

File: downstream/assemblies/platform/assembly-gw-resources.adoc (hunk 3 above)
Comment (dcdacosta) [reply downstream/assemblies/platform/assembly-gw-resources.adoc]: Fixed.

File: downstream/modules/platform/proc-gw-organizations-exec-env.adoc (hunk 4 above)
Comment (dcdacosta) [reply downstream/modules/platform/proc-gw-organizations-exec-env.adoc]: Fixed.

**Pull request merged**